### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by ghost** [dbitouze/french-tse#3]
If GUTenberg supports TeXnique, both logos could look alike to show the relationship between the website and the french TUG

**Comment by dbitouze** [comment]
I agree. Maybe we should wait for an exchange about the actual site's name: maybe better ideas than TeXnique will emerge...
